pyuvdata/uvdata/mwa_corr_fits.py

```diff
@@ -719,9 +759,34 @@ def read_mwa_corr_fits(
                 )
 
             if use_cotter_flags:
-                raise NotImplementedError(
-                    "reading in cotter flag files is not yet available"
+                # throw an error if matching files not submitted
```

Comment:
I think the error should move up here under this comment. That way you don't get the warning if it's just going to error anyway.